### ISSUE TITLE
What the Heck is Going On There?

### ISSUE THREAD

**Issue by DavidViral** [CSSEGISandData/COVID-19#1343]
Suddenly in the middle of a pandemic you change the structure of a database tracking the contagion!

I go for my evening update and I see missing data for March 23rd along with an announcement the DB is no longer supported,

The replacement database has no US state data,  And you broke my application - and for what?
Not backward compatible and I see no significant improvement structurally.

How many hours of our time to fix this?  And how long before you randomly change something again.

**Comment by yetzt** [comment]
Yeah, you nee to change to the new headers and aggregate states data before getting it consistant with yesterdays format. Already ranted about it in #1326

**Comment by ladris** [comment]
> Suddenly in the middle of a pandemic you change the structure of a database tracking the contagion!
> 
> I go for my evening update and I see missing data for March 23rd along with an announcement the DB is no longer supported,
> 
> The replacement database has no US state data, And you broke my application - and for what?
> Not backward compatible and I see no significant improvement structurally.
> 
> How many hours of our time to fix this? And how long before you randomly change something again.

Imagine, country state and city breakdown as well as FIPS information. You're right, completely useless and no gains whatsoever, other than the sheer volume of analytical data you can now process and statistically break down. Shame it broke your private unrelated database.

**Comment by imjoshellis** [comment]
I thought I was going crazy when I opened the dashboard I've been working on... Turns out, nope. They just decided to completely screw with their data structure with no announcement and no reasoning??  Ugh. Are there better, more reliable sources available (meaning sources that won't change their data on you out of nowhere)?

**Comment by cortical-iv** [comment]
> Imagine, country state and city breakdown as well as FIPS information. You're right, completely useless and no gains whatsoever, other than the sheer volume of analytical data you can now process and statistically break down. Shame it broke your private unrelated database.

You are right that there is new information that is helpful.

That said,  if in the middle of a pandemic, you make breaking changes to a db that hundreds of people are using to help others understand the data...is probably a Bad Idea.

**Comment by dhmacq** [comment]
> Suddenly in the middle of a pandemic you change the structure of a database tracking the contagion!
> 
> I go for my evening update and I see missing data for March 23rd along with an announcement the DB is no longer supported,
> 
> The replacement database has no US state data, And you broke my application - and for what?
> Not backward compatible and I see no significant improvement structurally.
> 
> How many hours of our time to fix this? And how long before you randomly change something again.

The changes were announced yesterday; there is no basis for being surprised.

The announcement stated that there would be new files for US state data. Oh, they're not there yet. Oh, but guess what, the US state data is still in the "old" files. I just looked in the "old" time series files, and yes, March 23 data is there. Yeah, they took out some zeros for some US entities, but they weren't real zeros, they were zeros that represented missing data. So they shouldn't have been there to begin with. Pretty trivial to fix with the software I use. 

The changes are not, repeat NOT, random. They were obviously done in response to other requests. Pay attention.

So, once they've set up a structure, it should be frozen for all time, just to avoid inconveniencing me, or you, or some other user? I don't think so.
